<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostDlg</class>
 <widget class="QDialog" name="PostDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="contentsBrowser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>371</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="titleBrowser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="authorBrowser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="commentsBrowser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>371</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除帖子</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>520</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改帖子</string>
   </property>
  </widget>
  <widget class="QPushButton" name="commentBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加评论</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="timeBrowser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="src.qrc">:/bkg5.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>contentsBrowser</zorder>
  <zorder>titleBrowser</zorder>
  <zorder>authorBrowser</zorder>
  <zorder>commentsBrowser</zorder>
  <zorder>deleteBtn</zorder>
  <zorder>modifyBtn</zorder>
  <zorder>commentBtn</zorder>
  <zorder>timeBrowser</zorder>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
